pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "open_gira"
+version = "0.0.1"
+authors = [
+  { name="Tom Russell", email="tom.russell@ouce.ox.ac.uk" },
+]
+description = "Helper package for open-gira workflow"
+readme = "README.md"
+requires-python = ">=3.9"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
```

Comment:
I don't know if we really are OS independent. There are plenty of linux commands used in shell blocks. Could these work in WSL?

Reply:
I think this metadata should apply for the package rather than the workflow as a whole, so should be easily portable.

Although, yes, workflow has been running fine in WSL, though I'm not sure if that counts as not-linux.